chat-plugins/mafia.js

```diff
@@ -1273,6 +1295,29 @@ exports.commands = {
 			}
 		},
 
+		role: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'mafia') return this.errorReply(`There is no game of mafia running in this room.`);
+			if (!(user.userid in room.game.players)) return false;
+			this.sendReplyBox(`Your role is: ${room.game.players[user.userid].role.safeName}`);
+		},
+
+		originalrolelist: 'rolelist',
+		orl: 'rolelist',
+		rl: 'rolelist',
+		rolelist: function (target, room, user, connection, cmd) {
+			if (!room.game || room.game.gameid !== 'mafia') return this.errorReply(`There is no game of mafia running in this room.`);
+			if (room.game.closedSetup) return this.errorReply(`You cannot show roles in a closed setup.`);
+			if (!this.runBroadcast()) return false;
+			const showOrl = (['orl', 'originalrolelist'].includes(cmd) || room.game.noReveal);
+			const roleString = (showOrl ? room.game.originalRoles : room.game.roles).sort((a, b) => {
+				if (a.alignment < b.alignement) return -1;
+				if (b.alignment < a.alignement) return 1;
+				return 0;
+			}).map((role) => { return role.safeName; }).join(', ');
+
+			this.sendReplyBox(`${showOrl ? `Original Rolelist: ` : `Rolelist: `}${roleString}`);
```

Comment:
We have a `game.roleString` and `game.originalRoleString`

Reply:
I'm reading off that because the colour for solo is unreadable on some of the backgrounds

Reply:
see prntscr.com/ivxvo4

Reply:
(I embedded your screenshot; in the future I recommend doing that to save a click)